MAINTAINING.md

```diff
@@ -1,8 +1,11 @@
 ### Deployment Checklist
 
+- `git checkout master && git pull upstream master`
 - `npm config get registry`
     - Make sure it's `https://registry.npmjs.org/`
 - `npm run clean`
+- optionally: `npm run nuke && npm run bootstrap`
+- `lerna exec -- npm prepublish`
```

Comment:
This one will go away (become part of bootstrap) with the [next version of Lerna](https://github.com/lerna/lerna/pull/250).